webhook-connector/shipment-events.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Shipment Events overview.
+---
+
+# Shipment Events
+
+When using the shipment webhook you may configure and receive below shipment events.
+
+## Shipment is issued or updated by the supplier
+
+The supplier can send a despatch advice or create or update a shipment in the portal.
+
+| ShipmentEvent                  | Webhook Configuration |
+| ------------------------------ | --------------------- |
+| `ShipmentIssuedBySupplier`     | New shipment is issued by the supplier |
```

Comment:
These are the literal texts from the portal, eg. the webhook configuration in the portal.

Reply:
Same as earlier.

Reply:
The shipment is issued according to our shipment process.
This can be done either by sending a dispatch advice or via the portal, which is stated in the sentence above.